README.md

```diff
@@ -54,7 +54,7 @@ A set of [React](https://reactjs.org/) components created by, and for, [Mavenlin
 
 - Use in your project
 
-  ```jsx
+  ```js
```

Comment:
Why are we swapping all of our `jsx` for `js`?

Reply:
Yeah! I want to keep it as simple and consistent with React Styleguidist docs.